azext_edge/edge/commands_dataflow.py

```diff
@@ -4,24 +4,91 @@
 # Licensed under the MIT License. See License file in the project root for license information.
 # ----------------------------------------------------------------------------------------------
 
-from typing import Iterable
+from typing import Iterable, Optional
 
 from .providers.orchestration.resources import DataFlowEndpoints, DataFlowProfiles
 
 
-def show_dataflow_profile(cmd, profile_name: str, instance_name: str, resource_group_name: str) -> dict:
+def create_dataflow_profile(
+    cmd,
+    profile_name: str,
+    instance_name: str,
+    resource_group_name: str,
+    profile_instances: Optional[int] = 1,
```

Comment:
when it has a default value, no need for the `Optional`, for example

profile_instances: int = 1,
log_level: str = "info",